### PR TITLE
setup for Datasette service [FOOD-76] [FOOD-63] [FOOD-64] [FOOD-98]

### PR DESCRIPTION
**Pull request by hotzevzl**
This PR allows to spin up a Datasette instance with Foodscapes tabular data via the single Docker Compose setup for the whole set of services.

It also includes the change needed to let the Docker build process for the container images of both the TiTiler and Datasette services retrieve and bake the source data (COG file and SQLite database) into the respective container images.

And finally, I sneaked in a small change to allow easily configuring tiler and datasette services to be served behind a reverse proxy, as we need it for the initial dev environment setup (and this could possibly be useful in staging/production depending on the specific setup we decide on).

## How to test

- Ideally, make sure your local Docker environment doesn't have any traces of previous Foodscapes Docker setup
- Make sure your Docker is up to date (see main `README.md` about minimum recommended version and specific setting to activate/update in case BuildKit builds were not enabled by default in previous local installs of Docker)
- Following the _Quick start_ section of the `README` should build and start the client app, the TiTiler service and Datasette

For the scope of this PR, Datasette should then be available at http://localhost:3203 (if using the default port settings from `env.default` copied over to `.env`.

### CODE REVIEW

File: data/import/foodscapes/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Creating a SQLite db from Foodscapes CSV data
+
+This should be set up to run in a Docker container: as a first quick step,
+however, the following steps should be all that is needed. The only dependency
+is the `sqlite3` binary.
+
+```
+sudo apt install sqlite3 # or whatever appropriate to install sqlite3 on your OS
```
Comment (BielStela) [inline data/import/foodscapes/README.md:8]: I will add this to make it super straightforward dummy proven to setup :sweat_smile: 
```suggestion
sudo apt install sqlite3 # or whatever appropriate to install sqlite3 on your OS
mkdir source
mkdir dest
```

File: data/import/foodscapes/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+cp /path/to/local/copy/of/foodscapes.csv ./source
+sh foodscapes_csv-to-sqlite.sh
+```
+
+The Foodscapes data will be available as `./dest/foodscapes.db`.
+
+Indexes are currently added to each column: hence a very large file size,
+compared to the source CSV file.
```
Comment (BielStela) [inline data/import/foodscapes/README.md:16]: After creating the sqlite file, I have to copy/move it to `datassette/data` no? If so, maybe note it down here too would be nice

File: ENV_VARS.md [hunk 3]
```diff
@@ -21,9 +21,95 @@ file.
...
+### Checksums for source data
+
+COG and tabular data are added to the TiTiler and Datasette service container
+images, respectively, fetching them from a remote location through an `ADD`
+Docker build step.
+
+In order to make sure that the files being added are the ones expected (i.e.
+they are the exact revision of the data that is needed, they have not been
+tampered with - either accidentally or maliciously, etc.), a checksum is
+calculated by the Docker build process and this is compared to the checksum
```
Comment (BielStela) [inline ENV_VARS.md:111]: love this

File: data/import/foodscapes/README.md (hunk 1 above)
Comment (hotzevzl) [reply data/import/foodscapes/README.md:8]: good point. I have opted for pre-creating the folders (fd8889e67b92d6cbe18950bde2eee6cab0f24717) while keeping them gitignored: I like having the directory structure ready and visible, in fact 😄

File: data/import/foodscapes/README.md (hunk 2 above)
Comment (hotzevzl) [reply data/import/foodscapes/README.md:16]: right. it's even a bit more complicated than that, so I documented a couple of options in 43f2a90b76cddeee7137fb5fbe918850771b3a4b

File: ENV_VARS.md (hunk 3 above)
Comment (hotzevzl) [reply ENV_VARS.md:111]: you can see why I wear tin foil hats all the time 😝

**Review by BielStela — APPROVED** [review]
Looks good to me

**Pull request merged**